example/eblob_backend.c

```diff
@@ -1078,6 +1080,10 @@ static int blob_send(struct eblob_backend_config *cfg, void *state, struct dnet_
 		if (err)
 			goto err_out_send_fail_reply;
 
+		/* stop if an error is occurred during write */
+		if (ctl->write_error)
+			goto err_out_put;
+
```

Comment:
I had missed iterator, so I have added the same check to `dnet_iterator_callback_server_send`, now it will stop iterator if some write will be failed.